### PR TITLE
Adding brief section on callbacks - not sure if this is a good idea

### PR DESCRIPTION
**Pull request by sansaid**
* Adding section on callbacks - not sure if this is a good idea yet, but thought I'd put it in the syllabus to see how it "feels" with the rest. Thoughts?

### CODE REVIEW

File: js-core-1/week-3/lesson.md [hunk 1]
```diff
@@ -15,11 +16,49 @@
 By the end of this class, you should be able to answer these questions:
 
 * What is a callback and how do you use it?
+* What are anonymous functions?
 * How do you use array methods and properties to manipulate arrays?
 * How can you chain array methods?
 
 ---
 
+## Callbacks
```
Comment (jamesgorrie) [inline js-core-1/week-3/lesson.md:25]: @sansaid I think this is a great idea.

We could start with explaining functions as values, as this is integral - and a way to use functions as values is callbacks.

What do you think?

**Comment by jamesgorrie** [comment]
I think this is a great idea, I think the order in which things are introduced is a bit odd in the syllabus. We don't need to go into much depth, but something like this at least give the student and anchor as to why passing functions about is OK and they will be practising it a lot outside of the theory as it's just a javascripty thing.

**Review by ismaelgt — COMMENTED** [review]
There is already a subsection on Callbacks under Array map

**Comment by sansaid** [comment]
Ah, that's an oversight! Thanks @ismaelgt ! Should we keep it where it is or move it to its own section? We can keep it as it is, but I was thinking about teaching it in the start to the students anyway. Thoughts? @aldeez988 , @jamesgorrie ?

**Review by aldeez988 — APPROVED** [review]

**Comment by sansaid** [comment]
Closing PR as callbacks section already added

**Pull request closed without merging**